bot/src/db.js

```diff
@@ -60,3 +60,17 @@ export async function archive(tags, status, ts) {
     [tags, status, ts]
   )
 }
+
+export async function setFrequency(tagsWithFrequency, ts) {
+  const uniqueTags = {}
+  tagsWithFrequency.forEach(([tag, frequency]) => {uniqueTags[tag] = frequency})
```

Comment:
The command can contain a tag multiple times, but the query can't touch the same row twice.
Could be done on server side, except aggregation function on frequency in group by would have to be min or max, which is kind of arbitrary.

Reply:
So.. Why don't we do this right away? :slightly_smiling_face: 

i.e. Instead of building an array for  `tagsWithFrequency`, you could make it an object, which is passed to `setFrequency`, whose keys are the tags and values are the frequencies.